### DISCUSSION TITLE
机器通知里的字体颜色如何配置？

### DISCUSSION THREAD

**Discussion by AiChatD** [dqzboy/DevOps#2] [Q&A] [answered]
你好，我想问下，机器人里面的字体颜色是怎么实现的？

**Comment by dqzboy** [comment, marked as answer]
在markdown 中使用font标签，具体可以查看钉钉官方文档[marathon语法](https://open.dingtalk.com/document/orgapp/enterprise-internal-robots-send-markdown-messages#title-w87-omz-3es)
